socorro/external/postgresql/adi.py

```diff
@@ -23,14 +23,16 @@ def get(self, **kwargs):
             ('start_date', yesterday, 'date'),
             ('end_date', tomorrow, 'date'),
             ('product', '', 'str'),
-            ('version', '', 'str'),
+            ('versions', [], 'list'),
+            ('platforms', [], 'list'),
```

Comment:
I do not really like the ``s`` at the end of parameters anymore. But maybe it makes more sense here. In the SuperSearch world everything can be a list so it's a bother to make everything plural. 

I guess the actual question is: do we want to be consistent with the SuperSearch API? In which case we should not add ``s`` to parameters.